elements/rh-secondary-nav/rh-secondary-nav.css

```diff
@@ -1,20 +1,21 @@
 :host {
-  --_chevron-size: 0.35em;
-  --_chevron-thickness: 0.125em;
-  --_chevron-up-angle: 45deg;
-  --_chevron-down-angle: -135deg;
+  --_chevron-size: calc(var(--rh-font-size-body-text-md, 1rem) * 0.35);
+  --_chevron-thickness: calc(var(--rh-font-size-body-text-md, 1rem) * 0.125);
+  --_chevron-up: 45deg;
+  --_chevron-down: -135deg;
+  --_chevron-correction-x: calc(-1 * var(--rh-font-size-body-text-md, 1rem) / 16);
+  --_chevron-correction-y: calc(-1 * var(--rh-font-size-body-text-md, 1rem) / 16);
   --_chevron-color: var(--rh-color-text-primary-on-light, #151515);
-  --_chevron-transform-collapsed:
-    rotate(var(--_chevron-up-angle))
-    translate(calc(-1 * var(--_chevron-size)), var(--_chevron-thickness));
-  --_chevron-transform-expanded:
-    rotate(var(--_chevron-down-angle))
-    translate(var(--_chevron-thickness), calc(var(--_chevron-inverse, -1) * var(--_chevron-size)));
+  --_chevron-transform-collapsed: 
+      rotate(var(--_chevron-up)) 
```

Comment:
can we use transform properties for these? 
in otherwords `rotate:` and `translate:` instead of `transform:`? maybe that's out of scope. these are `--_private` so we can always make it a `refactor(secondary-nav)` later on

Reply:
Let's revisit this after.